addons/web/static/src/views/fields/progress_bar/progress_bar_field.scss

```diff
@@ -28,6 +28,7 @@
     .o_progressbar {
         // Force progress bars to respect table's layout
         display: table-row;
+        justify-content: end;
     }
```

Comment:
```suggestion
    .o_progressbar {
        // Force progress bars to respect table's layout
        display: table-row;
    }
    
    .o_progress {
        flex-grow: 1;
    }

    .o_progressbar_value {
        width: 45px;
    }
```
@nle-odoo what do you think about using `flex-grow: 1` but setting a fixed width to the percentage that way all progress bars can grow to the same width regardless of the actual percentage value. 

<img width="1117" alt="Screen Shot 2023-11-24 at 8 59 28 AM" src="https://github.com/odoo/odoo/assets/61014499/86d02008-de4e-44e6-8a54-eed88562a7c9">

Reply:
this might be ok too (this probably should be right aligned still (I guess it's the remaining space from 45px that we see)

but the current PR is ok for me too, the CSS specify that the bar is 100px so it was possibly intended